source/elements/oneMKL/source/domains/dft/config_params/workspace_placement.rst

```diff
@@ -0,0 +1,86 @@
+.. SPDX-FileCopyrightText: Codeplay Software
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_dft_config_workspace_placement:
+
+Workspace placement
+--------------------------------------
+
+DFT implementations often require temporary storage for intermediate data whilst computing DFTs.
+This temporary storage is referred to as a *workspace*.
+Whilst this is managed automatically by default (``config_param::WORKSPACE_AUTOMATIC``), 
+it may be preferable to provide an external workspace (``config_param::WORKSPACE_EXTERNAL``) for the following reasons:
+
+* To reduce the number of GPU mallocs / frees
+* To reduce memory consumption
```

Comment:
Following the discussion in https://github.com/oneapi-src/oneAPI-spec/pull/509#discussion_r1381333385 we could mention here that using `WORKSPACE_EXTERNAL` may hinder performance for some backends and configurations.

Reply:
Done in [c75415e](https://github.com/oneapi-src/oneAPI-spec/pull/509/commits/c75415e9b9d3edeade96de944e76ead7e94d3479).